Orange/widgets/tests/test_workflows.py

```diff
@@ -0,0 +1,33 @@
+from os import listdir
+from os.path import isfile, join, dirname, exists
+
+from Orange.canvas.application import workflows
+from Orange.canvas.scheme import widgetsscheme
+from Orange.canvas.scheme.readwrite import scheme_load
+from Orange.widgets.tests.base import WidgetTest
+from Orange.widgets import tests
+
+class TestWorkflows(WidgetTest):
+    def test_scheme_examples(self):
+        """
+        Test if Orange workflow examples can be opened. Examples in canvas
+        and also those placed "workflows" subfolder.
+        GH-2240
+        """
+        def test(filename):
```

Comment:
convert this function to a (static) method on the class. name it something along the lines of "assertCanBeLoaded". 

Add some error handling code to the test that would catch the error and fail the test with the name of the workflow that could not be loaded. When the tests fails, the name of the problematic workflow will be much more useful than the actual exception (as it will allow us to run and debug the problematic workflow).